motion/thruster_interface_auv_cpp/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.5)
+project(thruster_interface_auv_cpp)
+
+# Required dependencies
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(std_msgs REQUIRED)
+find_package(vortex_msgs REQUIRED)
+
+# Include directories
+include_directories(include/${PROJECT_NAME})
```

Comment:
This should just be include_directories(include)
Use relative path when including the header files of this project